### PR TITLE
deploy(vars): adjust new layout approach 

### PR DESCRIPTION
**Pull request by maria-Petrova**
The PR contains changes to adjust the alternative layout approach for the upcoming releases and modulefile updates.

### CODE REVIEW

**Comment by maria-Petrova** [comment]
DAALROOT will be removed in a separate PR.

**Comment by maria-Petrova** [comment]
[Pre-commit](http://intel-ci.intel.com/ee314c82-a54a-f12c-b6b4-a4bf010d0e2e)

**Comment by maria-Petrova** [comment]
[Pre-commit](http://intel-ci.intel.com/ee4358d6-b011-f1cb-92b2-a4bf010d0e2e)

File: deploy/local/dal [hunk 1]
```diff
@@ -26,79 +26,48 @@ if { $tcl_version < $min_tcl_ver } {
...
-
-module-whatis "Intel(R) oneAPI Data Analytics Library for $daal_target_arch."
+set dalroot "$componentroot"
+set daal_target_arch "intel64"
 
 # Setup environment variables
-setenv          DAL_MAJOR_BINARY   __DAL_MAJOR_BINARY__
-setenv          DAL_MINOR_BINARY   __DAL_MINOR_BINARY__
-setenv          DALROOT            $daalroot
-setenv          DAALROOT           $daalroot
```
Comment (napetrov) [inline deploy/local/dal:97]: We still have DAALROOT used in scikit, so we have to make changes there as well

https://github.com/search?q=repo%3Aintel%2Fscikit-learn-intelex%20DAALROOT&type=code

+ Probably check other places like automation, installer validation

**Review by homksei — APPROVED** [review]

**Review by napetrov — APPROVED** [review]

**Pull request merged**